pyiron/sphinx/base.py

```diff
@@ -479,9 +704,7 @@ def from_hdf(self, hdf=None, group_name=None):
     def from_directory(self, directory):
         try:
             if not self.status.finished:
-                subprocess.call(
-                    "module load sphinx && sx2aims", cwd=directory, shell=True
-                )
+                os.system("module load sphinx && sx2aims")
```

Comment:
The module load part is specific to our cluster - I know it was in there before, but it should not be. The way we call executables is using the scripts in https://github.com/pyiron/pyiron-resources/tree/master/sphinx/bin - so maybe we could have a sx2aims script in there and then have the module load command in there if necessary.

Reply:
Good point- in this case I think we can eliminate that whole code block. As far as I can tell, the conversion to FHIaims was just to be able to read in the structure via ase_to_pyiron. Sam already had a function to read sphinx files directly

Reply:
Maybe we should move this discussion to a separate pull request. I have to admit I was not even aware of this part.

Reply:
Sure no problem- @samwaseda might have further thoughts on this. It's not really related to input parsing and I also wouldn't have noticed it if it hadn't broken the checks.